### PR TITLE
2016-11-28 - Last week in ManageIQ - Fixing bugs and fixing bugs

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

**Comment by JPrause** [comment]
Thanks @skateman 
@bronaghs @d-m-u @imtayadeway @syncrou can each of you review David's blog entry.

**Comment by d-m-u** [comment]
Looks great.

**Comment by JPrause** [comment]
LGTM :+1:

File: site/_posts/2016-11-28-fixing-bugs.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+But that is not all, he also added support for provider policies in our
+[User Interface](https://github.com/ManageIQ/manageiq/pull/11002).
+
+## Deleted
+
+[Libor](https://github.com/lpichler) found out that the *show* and *list*
+permissions for Tenants have the same purpose and unified them under a [new
+permission](https://github.com/ManageIQ/manageiq/pull/12844) called *view*.
+
+As you know, we introduced a new version timelines a few weeks ago. The old
```
Comment (syncrou) [inline site/_posts/2016-11-28-fixing-bugs.md]: @skateman - possibly missing an `of` between version and timelines?

**Comment by syncrou** [comment]
@skateman - I really enjoyed the writing! - I have a couple of small comments, though I could have easily missed the context.  Minus answers to those - 👍 LGTM.

File: site/_posts/2016-11-28-fixing-bugs.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+title: "Last Week in ManageIQ: Fixing bugs and fixing bugs"
```
Comment (imtayadeway) [inline site/_posts/2016-11-28-fixing-bugs.md]: :heart:

File: site/_posts/2016-11-28-fixing-bugs.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+We are still focused on the stabilization of the upcoming Euwe release.
+In numbers this means 91 commits merged from 88 PRs created by 40 of our
+contributors since last week. If you are interested, check out the list
+of [open issues]((https://github.com/manageiq/manageiq/issues)) to help
+us in increasing these numbers.
+
+## Improved
+
+Chart widgets on the dashboard are sometimes displaying too many information
```
Comment (imtayadeway) [inline site/_posts/2016-11-28-fixing-bugs.md]: s/many/much/

File: site/_posts/2016-11-28-fixing-bugs.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+on a too tiny place, making some data unreadable. There was a primitive
+zooming feature available, but it was not always helpful. Now thanks to
+our charts expert [Robin](https://github.com/PanSpagetka) the zoomed version
+uses [a full screen modal](https://github.com/ManageIQ/manageiq/pull/12811).
```
Comment (imtayadeway) [inline site/_posts/2016-11-28-fixing-bugs.md]: No strong feelings here, but do you have any thoughts on including something pictorial in the blog post body, like the before/after shots in the PR? A picture can paint a thousand words! Throwing this open as a general question to all reviewers.

File: site/_posts/2016-11-28-fixing-bugs.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+There were also some improvements in the Automate part of our codebase.
+For example [Patrik](https://github.com/pkomanek) continued the [refactoring
+of automate methods](https://github.com/ManageIQ/manageiq/pull/12523). His
+victim was the `available_flavors` method for `miq_ae_orchestration`.
+
+## Fixed
+
+In these though days almost half of the number of PRs have the word "fix"
+in their title. This means it is extremely hard to select which one to
```
Comment (imtayadeway) [inline site/_posts/2016-11-28-fixing-bugs.md]: possibly s/one/ones/ ?

Of course, there is no limit to how many fixes you can to cover here :wink:

File: site/_posts/2016-11-28-fixing-bugs.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+present here.
+
+We had some issues with infinite spinners on the Reports page when logged
+in as a non-admin user. The problem was with tree unlocking after an AJAX
+change on the page. Thanks to our [Happy Martin](https://github.com/himdel)
+it is now [fixed](https://github.com/ManageIQ/manageiq/pull/12846).
+
+The select box for policy profile selection on the policy simulation screen
+had a fixed size and therefore, it could not properly display longer strings.
+The cleverness of [Milan](https://github.com/mzazrivec) came handy now, as he
```
Comment (imtayadeway) [inline site/_posts/2016-11-28-fixing-bugs.md]: Wondering if s/came handy now/came in handy/ ? Though I am not sure if that is just a Britishism :wink:

**Comment by imtayadeway** [comment]
@skateman great post! Just a couple of things above

File: site/_posts/2016-11-28-fixing-bugs.md (hunk 4 above)
Comment (skateman) [inline site/_posts/2016-11-28-fixing-bugs.md]: I would just add one screenshot, probably the final result.

File: site/_posts/2016-11-28-fixing-bugs.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+uses [a full screen modal](https://github.com/ManageIQ/manageiq/pull/12811).
+
+There were also some improvements in the Automate part of our codebase.
+For example [Patrik](https://github.com/pkomanek) continued the [refactoring
+of automate methods](https://github.com/ManageIQ/manageiq/pull/12523). His
+victim was the `available_flavors` method for `miq_ae_orchestration`.
+
+## Fixed
+
+In these though days almost half of the number of PRs have the word "fix"
```
Comment (skateman) [inline site/_posts/2016-11-28-fixing-bugs.md]: Typo `tough`, yes!

**Comment by skateman** [comment]
@imtayadeway fixed, thanks!

**Comment by JPrause** [comment]
Cool! Thanks all! @hayesr please merge.

**Pull request merged**